frontend/src/components/managePermissions/table/ApplicationAdminTable.vue

```diff
@@ -47,19 +47,6 @@ const newAppAdminIds = computed(() => {
 
 const adminFilters = ref({
     global: { value: '', matchMode: FilterMatchMode.CONTAINS },
-    'user.user_name': { value: null, matchMode: FilterMatchMode.STARTS_WITH },
```

Comment:
Realized we don't need these filters, those are for individual column filter. But in our case, we just need to global filter.

Reply:
That's great, thanks Catherine!